changelog/8.8.0.yaml

```diff
@@ -0,0 +1,287 @@
+        checksum: ff3bd63aa3dfa8d9c7d8f708cf7d3febde4e5bae
+    - kind: bug-fix
+      summary: Add local-config.yaml to diagnostics and merge together in inspect.
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2529
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2390
+      timestamp: 1682005908
+      file:
+        name: 1682005908-Add-local-config.yaml-to-diagnostics-and-merge-together-in-inspect..yaml
+        checksum: 62a404c9dc8b45aa3d17042fa482545709ce1ac2
+    - kind: enhancement
+      summary: Increase timeout, add config for timeout in fleet setup
+      description: ""
+      component: config-manager
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2541
+      issue: []
+      timestamp: 1682117462
+      file:
+        name: 1682117462-fleet-connection-changes.yaml
+        checksum: e4acb525c2183d0e69eabcb6af48ebb14ce0e9e3
+    - kind: bug-fix
+      summary: fix pprof diagnostics
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2617
+        - https://github.com/elastic/elastic-agent/pull/2549
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2530
+      timestamp: 1682442537
+      file:
+        name: 1682442537-fix-pprof-diagnostics.yaml
+        checksum: 403815e914a64e726360f3a2219547faf9b8ec80
+    - kind: feature
+      summary: Adding Cloud Defend application to Elastic Agent
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2617
+        - https://github.com/elastic/elastic-agent/pull/2551
+      issue: []
+      timestamp: 1682455535
+      file:
+        name: 1682455535-Adding-Cloud-Defend-application-to-Elastic-Agent.yaml
+        checksum: e2c53fe2bb4dec679c2b8be1e858f02843e69538
+    - kind: bug-fix
+      summary: Fix container service token path issue
+      description: |
+        Fix issue that occurs when specifing a FLEET_SERVER_SERVICE_TOKEN_PATH with
+        the agent running in a Docker container where both the token value and path
+        are passed in the enroll section of the agent setup.
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2576
+      issue: []
+      timestamp: 1682953274
+      file:
+        name: 1682953274-Fix-container-service-token-path-issue.yaml
+        checksum: 2434964330fe5268c12b60c13db26064f4678bf3
+    - kind: bug-fix
+      summary: Create install marker file during upgrade from versions < 8.8.0
+      description: ""
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2655
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2645
+      timestamp: 1683748237
+      file:
+        name: 1683748237-upgrade-install-marker.yaml
+        checksum: b395497cc60dda4280483d5eaa6eaad7855223b0
```

Comment:
According to [Craig's comment](https://github.com/elastic/ingest-docs/pull/219#discussion_r1198284372), `We hid this feature so we don't need this changelog entry yet. This is also a fix for a bug that was never released.`